aiven_mysql_migrate/utils.py

```diff
@@ -52,6 +54,9 @@ def from_uri(uri: str, name: Optional[str] = None):
         except ValueError as e:
             raise WrongMigrationConfigurationException(f"{uri!r} invalid port") from e
 
+        if len(res.password) > MAX_PASSWORD_LENGTH:
```

Comment:
MySQL is using C's `strlen`, the password `'é' * 17` is refused by MySQL, this should use `len(res.password.encode())`.